besu/terraform/aws/ibftScale/ibft4caliper/variables.tf

```diff
@@ -13,9 +13,9 @@ variable "caliper_repo" {
   default = "https://github.com/benjamincburns/caliper"
 }
 
-variable "caliper_repo" {
-  default = "https://github.com/hyperledger/caliper"
-}
+# variable "caliper_repo" {
+#   default = "https://github.com/hyperledger/caliper"
+# }
```

Comment:
why is this commented out? suggest adding explanation or removing

Reply:
I think it's worth leaving this in as commented - maybe just include a note that this is the upstream caliper repo